client/src/components/achievements/achievementsOnboarding.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable react/button-has-type */
+/* add new, delete, edit */
```

Comment:
Add file docs above to describe what this Component does

Reply:
especially since you have multiple components exported in this file, it'd be useful to document all of them